src/server/game/Server/Packets/LootPackets.cpp

```diff
@@ -34,7 +34,7 @@ WorldPacket const* WorldPackets::Loot::LootResponse::Write()
     _worldPacket << uint32(Items.size());
     _worldPacket << uint32(Currencies.size());
 
-    for (LootItemData const& item : Items)
+    for (auto const& item : Items)
```

Comment:
You don't have to change this but avoid doing these sort of replacements.
`auto` works great for some types (e.g iterators) but it removes a lot of readability from the code